internal/pkg/handler/updated-handler.go

```diff
@@ -20,11 +32,210 @@ func (r ResourceUpdatedHandler) Handle() error {
 		// process resource based on its type
 		if _, ok := r.Resource.(*v1.ConfigMap); ok {
 			logrus.Infof("Performing 'Updated' action for resource of type 'configmap'")
+			rollingUpgrade(r, "configmaps", "deployments")
```

Comment:
Whats the point of checking for resource type here and passing it to rollingUpgrade and then checking for resource type again in that function? Seems pretty redundant